alacritty/src/display.rs

```diff
@@ -579,18 +579,20 @@ impl Display {
 
         self.draw_render_timer(config, &size_info);
 
-        // Handle search bar rendering.
+        // Handle regex search.
```

Comment:
```suggestion
        // Handle regex search and compute IME position.
```

I feel like it reflects things a bit more?